### ISSUE TITLE
Animation in meterGauge chart

### ISSUE THREAD

**Issue by gelo32k** [svandecappelle/jQPlot#12]
I would like to request an enhancement to the existing meterGauge chart.

The desired enhancement is a needle animation effect similar to this: http://bl.ocks.org/msqr/3202712

It doesn't have to be exactly like that, just a simple animation to give the experience of a moving needle as the values increases.

Thanks.

**Comment by svandecappelle** [comment]
Good idea ! 
Planned into 1.0.10.

**Comment by johanbove** [comment]
Cool idea! Could use something like this as well.
Will D3.js become a dependency for jqPlot for this feature?

**Comment by gelo32k** [comment]
I don't think so. The bar chart already has animation without dependency on D3.js.

**Comment by svandecappelle** [comment]
D3 is a svg building library.
JQplot builds HTML5 canvas.

This is two different way to builds charts on the web.
If we would like to integrate D3 features, we need to rewrite a new build system using SVG.

**Comment by vkaranat** [comment]
Since this is an old post.. Has this already been implemented ? 
how can we animate the needle in the meter gauge. 
Dint find any reference about it online..

**Comment by svandecappelle** [comment]
Not yet implemented because of complexity to rebuild chart on each animation.
Canvas animations needs to rebuild a location on the canvas chart. Doing that means that cleared region need to be rebuilt. My try to animate it causes performances issues.

I study an alternative to this point.
